types/src/index.ts

```diff
@@ -50,5 +51,7 @@ export {
   CaliperGradingProfile,
   CaliperEnvelope,
   IPayload,
+  AssignedCondition,
```

Comment:
Don't export `AssignedCondition` use it like this
`IExperimentAssignmentv5.AssignedCondition[number]`

Because each Experiment Assignment can have it's own type of `AssignedCondition`

Reply:
code updated